Time,Test Feature,Status,Comments
16:13:12,,,Begin executing Suite: Default Suite
16:13:12,,,Begin executing Test: TestArtifact
16:13:12,,,Before Invocation of method: setUp
16:13:12,,,After Invocation of method: setUp
16:13:12,,,Test Status: extendBaseFlow
16:13:12,,,Before Invocation of method: extendBaseFlow
16:13:19,,,Url is https://qa.com/
16:13:19,,,Environment is QA and sponsor not specified
16:13:19,,,credential match WITHOUT tier OR sponsor info provided
16:13:19,,,username is  and password is 
16:13:19,extendBaseFlow,,After Invocation of method: extendBaseFlow
16:13:19,extendBaseFlow,fail,Test Status: extendBaseFlow
16:13:19,extendBaseFlow,fail,Test Context: org.testng.TestRunner@555cf22
16:13:19,extendBaseFlow,fail,Test Throwable: java.lang.AssertionError: expected [true] but found [false]
,,,	at org.testng.Assert.fail(Assert.java:96)
,,,	at org.testng.Assert.failNotEquals(Assert.java:776)
,,,	at org.testng.Assert.assertTrue(Assert.java:44)
,,,	at org.testng.Assert.assertTrue(Assert.java:54)
,,,	at tests.ExtendBase.extendBaseFlow(ExtendBase.java:20)
,,,	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
,,,	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
,,,	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
,,,	at java.lang.reflect.Method.invoke(Method.java:498)
,,,	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
,,,	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
,,,	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
,,,	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
,,,	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
,,,	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
,,,	at org.testng.TestRunner.privateRun(TestRunner.java:648)
,,,	at org.testng.TestRunner.run(TestRunner.java:505)
,,,	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
,,,	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
,,,	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
,,,	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
,,,	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
,,,	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
,,,	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
,,,	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
,,,	at org.testng.TestNG.runSuites(TestNG.java:1049)
,,,	at org.testng.TestNG.run(TestNG.java:1017)
,,,	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
,,,	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
,,,
16:13:19,,,Before Invocation of method: wrapUp
